src/App.css

```diff
@@ -36,3 +36,9 @@
     transform: rotate(360deg);
   }
 }
+
+.Review-picture{
+  width:5vw;
```

Comment:
vw is maybe not the best measurement for images as it's based on the viewport - 5vw is very small as it's only 5% of the viewport width